src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -44,6 +44,9 @@ public sealed class Cookie
         // Space (' ') should be reserved as well per RFCs, but major web browsers support it and some web sites use it - so we support it too
         private static readonly IndexOfAnyValues<char> s_reservedToNameChars = IndexOfAnyValues.Create("\t\r\n=;,");
 
+        private static readonly IndexOfAnyValues<char> s_domainChars =
+            IndexOfAnyValues.Create("-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz");
```

Comment:
> Do you think something will be helpful?

I don't want to add something like that which encourages unnecessary cost (both the Create and Include calls would do all the work to construct the instance).  Instead, if someone is using a subset repeatedly and wants to avoid the repetition, they can define a const:
```C#
internal const string AsciiLetterAndDigits = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
```
with whatever it is they're repeatedly using, and then use const string folding:
```C#
IndexOfAnyValues.Create(AsciiLetterAndDigits + ".-_");
```
which will end up with the same outcome but no additional run-time cost.

Reply:
Sure, that works with strings / ROS\<char\> (and I picked a bad example).

Is there something similar that works with ROS\<byte\> too? 
Together with https://github.com/dotnet/aspnetcore/pull/45300#discussion_r1032955091 these are two points that could be improved for usability.

Note: I'm aware that this is an rather advance API and the cost to make the usage easier has to be set in relation to that.

Reply:
> Is there something similar that works with ROS<byte> too?

Today, no, but I'd rather see that addressed by improvements to the language.

> I'm aware that this is an rather advance API and the cost to make the usage easier has to be set in relation to that.

Yes :)